package.json

```diff
@@ -6,12 +6,12 @@
     "url": "https://github.com/openfoodfoundation/openfoodnetwork"
   },
   "devDependencies": {
-    "phantomjs-prebuilt": "~2.1.7",
-    "karma": "~0.13.22",
-    "karma-jasmine": "~0.3.8",
     "jasmine-core": "~2.4.1",
+    "karma": "^3.0.0",
+    "karma-coffee-preprocessor": "~0.3.0",
+    "karma-jasmine": "~0.3.8",
     "karma-phantomjs-launcher": "~1.0.0",
-    "karma-coffee-preprocessor": "~0.3.0"
+    "phantomjs-prebuilt": "~2.1.7"
```

Comment:
This must be exactly what was described in https://github.com/openfoodfoundation/openfoodnetwork/pull/3002#issuecomment-437234713. You can check @Matt-Yorkley 's answer to know how to get rid of them in this branch. Don't hesitate to reach us out on the #dev channel if you need help.

Reply:
Done!